### PR TITLE
Student assignments

### PR DESCRIPTION
**Pull request by Sapideh1071**
I have created a new student assignment page, and an index over the assignments a student has or has had

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  
+  def student_assignments
+  	@student = User.find(params[:id])
+  	@assignments = @student.assignments
+    puts "My assignments"
```
Comment (kat-h6) [inline app/controllers/users_controller.rb]: maybe you can take out these puts statements now :D

**Review by kat-h6 — CHANGES_REQUESTED** [review]
looks awesome! just that one thing, and I know you're still working on that show page

**Review by kat-h6 — COMMENTED** [review]
looks awesome!

**Comment by Sapideh1071** [comment]
I tried to pull the latest master, but I have a problem with the Gemlock
file. Second time this is happening.

tor. 10. jun. 2021 kl. 20:06 skrev Katharine Hao ***@***.***>:

**Pull request merged**